app/models/api/trainee_attributes/v01.rb

```diff
@@ -77,42 +77,45 @@ class V01
         EmailFormatValidator.new(record).validate
       end
 
-      def initialize(attributes = {})
-        attributes = attributes.to_h.with_indifferent_access
+      validates(:sex, inclusion: Hesa::CodeSets::Sexes::MAPPING.values, allow_blank: true)
```

Comment:
Shouldn't we validate for the keys?

```suggestion
      validates(:sex, inclusion: Hesa::CodeSets::Sexes::MAPPING.keys, allow_blank: true)
```

Reply:
it should be `.values`, it goes thro the mapper
https://github.com/DFE-Digital/register-trainee-teachers/blob/2d0987b253d15f311a42461c64ff082e88aed7d6/app/services/api/map_hesa_attributes/v01.rb#L67-L69

https://github.com/DFE-Digital/register-trainee-teachers/blob/794d45c1d380d787983315cede98d5eac0815f7f/app/lib/hesa/code_sets/sexes.rb#L7-L13

So by the time it gets thro the mapper to the trainee attributes is has been converted from a string to an integer.